pkg/services/sqlstore/migrations/accesscontrol/test/ac_test.go

```diff
@@ -103,121 +100,6 @@ func convertToRawPermissions(permissions []accesscontrol.Permission) []rawPermis
 	return raw
 }
 
-func TestMigrations(t *testing.T) {
```

Comment:
Looks like there are some leftover variables and functions that were only used by this block and can be removed